deployment/ansible/roles/raster-foundry.nexus/templates/nexus.conf.j2

```diff
@@ -0,0 +1,19 @@
+server {
```

Comment:
It may be worth bringing over the other recommended Nginx configuration settings from [here](https://help.sonatype.com/repomanager3/installation/run-behind-a-reverse-proxy#RunBehindaReverseProxy-nginx.1).

Namely:

```nginx
proxy_send_timeout 120;
proxy_read_timeout 300;
proxy_buffering    off;
keepalive_timeout  5 5;
tcp_nodelay        on;
```